src/core/feed.factory.ts

```diff
@@ -52,16 +52,26 @@ import {
 export class FeedFactory {
   constructor(private client: IgApiClient) {}
 
-  public accountFollowers(id?: string | number): AccountFollowersFeed {
-    const feed = new AccountFollowersFeed(this.client);
-    feed.id = id || this.client.state.cookieUserId;
-    return feed;
+  public accountFollowers(
+    id?: string | number,
+    options?: Partial<Pick<AccountFollowersFeed, 'searchSurface' | 'order' | 'query' | 'enableGroups'>>,
```

Comment:
Looks weird...
Let's make the only input arg of type `string | number | Partial<Pick<AccountFollowersFeed, 'id' | 'searchSurface' | 'order' | 'query' | 'enableGroups'>> ` ?